src/sdf_xarray/__init__.py

```diff
@@ -523,10 +553,43 @@ def guess_can_open(self, filename_or_obj):
 
 
 class SDFPreprocess:
-    """Preprocess SDF files for xarray ensuring matching job ids and sets time dimension"""
+    """Preprocess SDF files for xarray ensuring matching job ids and sets
+    time dimension.
 
-    def __init__(self):
+    This class is used as a 'preprocess' function within xr.open_mfdataset. It
```

Comment:
```suggestion
    This class is used as a 'preprocess' function within ``xr.open_mfdataset``. It
```

Or whatever you need to add to make this link to the Xarray docs...